theofidry/alice-data-fixtures/1.0/manifest.json

```diff
@@ -0,0 +1,5 @@
+{
+    "bundles": {
+        "Fidry\\AliceDataFixtures\\Bridge\\Symfony\\FidryAliceDataFixturesBundle": ["test"]
```

Comment:
I would also add `dev` like done in [nelmio/alice](https://github.com/symfony/recipes-contrib/blob/55858fa45cf9dadba4247b1d354e6ef5ffc6937f/nelmio/alice/3.2/manifest.json#L3)

Reply:
Added.